start_vscode_bessemer.sh

```diff
@@ -0,0 +1,478 @@
+#!/usr/bin/env bash
+
+###############################################################################
+#                                                                             #
+#  Script to run on a local computer to start a code-server on Bessemer and      #
+#  connect it with a local browser to it                                      #
+#                                                                             #
+#  Main author    : Samuel Fux                                                #
+#  Contributions  : Andreas Lugmayr                                           #
+#  Date           : October 2021                                              #
+#  Location       : ETH Zurich                                                #
+#  Version        : 0.1                                                       #
+#  Change history :                                                           #
+#                                                                             #
+#  28.10.2021    Initial version of the script based on Jupyter script        #
+#  Forked and edited for TUoS by J.Moore                                      #
```

Comment:
You may wish to add change history here.

Reply:
Will do. Thanks